holoviews/plotting/bokeh/element.py

```diff
@@ -2470,6 +2450,11 @@ class ColorbarPlot(ElementPlot):
         #FFFFFFFF or a length 3 or length 4 tuple specifying values in
         the range 0-1 or a named HTML color.""")
 
+    cticks = param.Parameter(default=None, doc="""
```

Comment:
Can we use `param.ClassSelector` here?

Even if it is verbose, it makes it easier to understand what is permitted.  You are also welcome to update `xticks` and `yticks` if they are also parameters.

Reply:
:| Very verbose indeed.